chia/wallet/wallet_state_manager.py

```diff
@@ -707,6 +715,8 @@ async def handle_cat(
         parent_coin_state: CoinState,
         coin_state: CoinState,
         coin_spend: CoinSpend,
+        peer: WSChiaConnection,
```

Comment:
Besides the fact that i think we probably shouldn't store the peer id, maybe just pass in `peer_id: bytes32` here.

Reply:
I like this because of how it matches the other similar functions like `handle_did`.  Passing in peer_id to one and a full peer into another seems more confusing than them all having the same API.

Reply:
This is now actually required to be a peer so this point is moot